custom_components/tuya_local/devices/alecoair_d16_dehumidifier.yaml

```diff
@@ -48,19 +48,19 @@ primary_entity:
     - id: 19
       name: error
       type: bitfield
-      # Docs mention E1 and E2, it isn't clear which is tank full, so for now
-      # treat both as that.
+      # E2 might be a compressor fault.
       mapping:
         - dps_val: 0
           value: OK
+          icon: "mdi:cup-outline"
```

Comment:
Don't set the icon here for the OK case, you want the preset icons to show through when the status is normal.